tools/distpartitioning/data_shuffle.py

```diff
@@ -175,7 +175,7 @@ def exchange_edge_data(rank, world_size, edge_data):
     edge_data.pop(constants.OWNER_PROCESS)
     return edge_data
 
-def exchange_node_features(rank, world_size, node_feature_tids, ntype_gnid_map, id_lookup, node_features):
+def exchange_features(rank, world_size, feature_tids, ntype_gnid_map, id_lookup, feature_data, feat_type, data):
```

Comment:
what's the last parameter for? `data`? is this really used below?

Reply:
Yes. it is used. This is for the edge_data case. Please check the condition where we check the data for the edges' case.